bin/reth/src/db/mod.rs

```diff
@@ -178,7 +178,6 @@ impl Command {
                     BlockTransitionIndex,
                     TxTransitionIndex,
                     SyncStage,
-                    TxHashNumber,
```

Comment:
nice catch,

opened this separately: https://github.com/paradigmxyz/reth/pull/1685